#! /bin/sh /usr/share/dpatch/dpatch-run
## 24_freecom_dvbt.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adds support for a freecom dvb-t card (LP: #152539).

@DPATCH@
diff -urNad lirc-0.8.3~/doc/lirc.hwdb lirc-0.8.3/doc/lirc.hwdb
--- lirc-0.8.3~/doc/lirc.hwdb	2008-05-10 16:47:09.000000000 -0500
+++ lirc-0.8.3/doc/lirc.hwdb	2008-05-10 16:47:55.000000000 -0500
@@ -73,6 +73,7 @@
 FlyVideo 98 (card=30);flyvideo;devinput;hw_default;life-view/lircd.conf.flyvideo;
 FlyVideo 98/FM /2000S (card=56);flyvideo;devinput;hw_default;life-view/lircd.conf.flyvideo;
 Flyvideo 98FM (LR50Q) / Typhoon TView TV/FM Tuner (card=36);flyvideo;devinput;hw_default;life-view/lircd.conf.flyvideo;
+Freecom DVB-T USB Stick;devinput;none;hw_default;freecom/lircd.conf.freeconf
 Hauppauge TV card;hauppauge;lirc_dev lirc_i2c;hw_default;hauppauge/lircd.conf.hauppauge;
 Hauppauge Nova-T 500;devinput;none;hw_devinput;hauppauge/lircd.conf.hauppauge_novat500;
 Hauppauge HVR-1300;hauppauge;lirc_dev lirc_i2c;hw_default;hauppauge/lircd.conf.hauppauge;
diff -urNad lirc-0.8.3~/remotes/freecom/lircd.conf.freeconf lirc-0.8.3/remotes/freecom/lircd.conf.freeconf
--- lirc-0.8.3~/remotes/freecom/lircd.conf.freeconf	1969-12-31 18:00:00.000000000 -0600
+++ lirc-0.8.3/remotes/freecom/lircd.conf.freeconf	2008-05-10 16:47:09.000000000 -0500
@@ -0,0 +1,35 @@
+#Freecom DVB-T Stick
+#By Nico Rdo
+
+begin remote
+  name Freecom_DVB-T_USB
+  bits 16
+  eps 30
+  aeps 100
+  one 0 0
+  zero 0 0
+  pre_data_bits 16
+  pre_data 0x8001
+  gap 237493
+  toggle_bit 0
+      begin codes
+          POWER 0x0074
+          MUTE 0x0071
+          0 0x000B
+          1 0x0002
+          2 0x0003
+          3 0x0004
+          4 0x0005
+          5 0x0006
+          6 0x0007
+          7 0x0008
+          8 0x0009
+          9 0x000A
+          VOL_UP 0x0073
+          VOL_DOWN 0x0072
+          CH_UP 0x0192
+          CH_DOWN 0x0193
+          ZOOM 0x0174
+          JUMP 0x0161
+      end codes
+end remote
